# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/canvas/ClientWebGLContext.cpp
# Commit: eef9a013f230
# Full Hash: eef9a013f230e145220accd1debebf37ef264891
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2024-07-25 09:31:33
# Description:
#   Bug 1908380 - Check for context loss in WebGLVertexArrayJS::WebGLVertexArrayJS. r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217634
# ==============================================================================

diff -r d1640e9f4325 -r eef9a013f230 dom/canvas/ClientWebGLContext.cpp
--- a/dom/canvas/ClientWebGLContext.cpp	Wed Jul 24 21:44:28 2024 +0000
+++ b/dom/canvas/ClientWebGLContext.cpp	Wed Jul 24 21:45:34 2024 +0000
@@ -6718,7 +6718,7 @@
 
 WebGLVertexArrayJS::WebGLVertexArrayJS(const ClientWebGLContext* const webgl)
     : webgl::ObjectJS(webgl),
-      mAttribBuffers(webgl ? webgl->Limits().maxVertexAttribs : 0) {}
+      mAttribBuffers(Context() ? Context()->Limits().maxVertexAttribs : 0) {}
 
 // -
 
